website/pages/std.md

```diff
@@ -3,20 +3,19 @@
 **This document is non-normative**.
 
 The VT6 protocol is structured into versioned **modules**.
-Before exchanging messages, servers and clients negotiate which VT6 modules, and versions thereof, they understand.
+Servers and clients negotiate which VT6 modules, and versions thereof, they understand.
```

Comment:
More like "Terminals and clients **can** negotiate...".

Reply:
I wanted to write **can** at first too, but found it too confusing in this particular place. Because visitors will read std.md before the specs and at that point the conveyed mental model should be simple, rather than precise. This document is non-normative anyway.

Will fix the "Servers" part though.

Reply:
ok